### DISCUSSION TITLE
Receiver's location

### DISCUSSION THREAD

**Discussion by Navids94** [geodynamics/sw4#202] [Q&A]
Does there exist a way to display the receiver's location that I used in the rec command to ensure that the receiver was located on soil or rock?
Thank you

**Comment by andersp** [comment]
One way is to enter the lat-lon coordinates into google earth.

**Reply by Navids94** [reply]
Thank you for your response. Since I use pfile on a Cartesian lattice and cartesian in the input file for the receiver location, the output file includes the following default target location (WGS84 longitude, latitude) (deg): -1.178460e+02 3.706163e+01.
